libs/sfm/bundler_common.cc

```diff
@@ -266,5 +266,50 @@ load_survey_from_file (std::string const& filename,
     in.close();
 }
 
+/* ---------------------- Feature undistortion -------------------- */
+
+namespace {
+    double distort_radius (double const r2, float const* params)
```

Comment:
Should this function explicitly take the undistortion parameters, i.e., `k0` and `k2`, or whatever it is? This way it's easier to understand and you may implicitly convert those to doubles (i.e., take `double k0` and `double k2` as parameters).

Reply:
changed